src/Lookup/RestrictedEntityLookup.php

```diff
@@ -105,6 +105,15 @@ public function getEntityAccessCount() {
 	}
 
 	/**
+	 * Resets the number of entities loaded via this object.
+	 *
+	 * @since 3.2.1
```

Comment:
I don't know how we should handle these situations. I would like to avoid having to locally apply patches to the build for minimal updates… and re-tagging a minimal 3.3.0 release also sounds awry to me.